rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -73,54 +127,18 @@
 _strict_mode = False
 
 
-def rerun_shutdown() -> None:
-    bindings.shutdown()
-
-
-atexit.register(rerun_shutdown)
-
-
-def unregister_shutdown() -> None:
-    atexit.unregister(rerun_shutdown)
-
-
-# -----------------------------------------------------------------------------
-
-
-def get_recording_id() -> Optional[str]:
-    """
-    Get the recording ID that this process is logging to, as a UUIDv4, if any.
-
-    You must have called [`rr.init`][] first in order to have an active recording.
-
-    The default recording_id is based on `multiprocessing.current_process().authkey`
-    which means that all processes spawned with `multiprocessing`
-    will have the same default recording_id.
-
-    If you are not using `multiprocessing` and still want several different Python
-    processes to log to the same Rerun instance (and be part of the same recording),
-    you will need to manually assign them all the same recording_id.
-    Any random UUIDv4 will work, or copy the recording id for the parent process.
-
-    Returns
-    -------
-    str
-        The recording ID that this process is logging to.
-
-    """
-    rid = bindings.get_recording_id()
-    if rid:
-        return str(rid)
-    return None
+# --- Init ---
 
 
 def init(
     application_id: str,
     recording_id: Optional[str] = None,
+    make_default: bool = True,
+    make_thread_default: bool = True,
     spawn: bool = False,
     default_enabled: bool = True,
     strict: bool = False,
-) -> None:
+) -> RecordingStream:
```

Comment:
Even though I think I'm the one that proposed this, I'm realizing this starts to conflate "init" with "RecordingStream::new".

I kind of feel like we want these to be 2 different APIs. Something like:
 - `init(...)` gets called once per app and always sets the default-recording.
 - `new_recording(...)` can be called many times and is used to create non-global recordings.

Reply:
fine by me